Makefile

```diff
@@ -261,6 +268,19 @@ ifeq ($(ENABLE_LIBYOSYS),1)
 TARGETS += libyosys.so
 endif
 
+ifeq ($(ENABLE_PYOSYS),1)
+	ifeq ($(PYTHON_MAJOR_VERSION),3)
+		LDLIBS += -lpython$(PYTHON_VERSION)m -lboost_python-py$(subst .,,$(PYTHON_VERSION)) -lboost_system -lboost_filesystem
+	else
+		LDLIBS += -lpython$(PYTHON_VERSION) -lboost_python-py$(subst .,,$(PYTHON_VERSION)) -lboost_system -lboost_filesystem
+	endif
+CXXFLAGS += -I/usr/include/python$(PYTHON_VERSION) -D WITH_PYTHON
```

Comment:
Trying to build on Arch Linux and I see this fail because the path is `/usr/include/python3.7m`. This should probably be using `python-config --includes` rather than hardcoded paths.

Reply:
I was unaware of python-config. Looks pretty userful!
cc270ea81bc7209e9e1673e6c3dbd5db6d3ceee8